services/blockchain-indexer/shared/utils/account.js

```diff
@@ -33,16 +34,11 @@ const getAccountsTable = () => getTableInstance(accountsTableSchema, MYSQL_ENDPO
 const getLisk32AddressFromPublicKey = publicKey => getLisk32AddressFromPublicKeyHelper(Buffer.from(publicKey, 'hex'));
 
 const updateAccountPublicKey = async (publicKey) => {
```

Comment:
This could be a one-liner.
```suggestion
const updateAccountPublicKey = async (publicKey) => indexAccountByPublicKey(publicKey);
```